### DISCUSSION TITLE
komutlar ve dosyalar

### DISCUSSION THREAD

**Discussion by ozkanseyyarer** [gusanmaz/BMB212-Algorithms-2223#4] [Q&A] [answered]
Eclipseden yapıp sonuçları buldum fakat aklıma takılan bir kaç soru var 
1-Github projemizde "student_10.csv" dosyaları oluşturmamız gerekiyor mu?
2-Eclipsde yaparken dosyanın direkt adını verip çalıştırıyorum projeme de öyle kaydettim
olması gereken bu mu -----> Student.readCSV("student_10.csv").length;

**Comment by gusanmaz** [comment, marked as answer]
Repolariniza data dosyalarini (CSV uzantili dosyalar) eklemenize gerek yok. Zaten kodlariniz bambaska (size verilmeyen) data dosyalari ile test edilebilir ama komut satirindan okunan argumani `ReadCSV` fonksiyonuna parametre olarak vererek Student dizisi elde edebiliyorsaniz bu durum bir sorun teskil etmiyor. 

Ilk 5 proje icin `Main.java` ,`Studenyt.java` ve `{ogrenci_no}.txt` ve `llm.txt` (bazi ozel durumlarda) dosyalari yeterli. Bu dosyalarin hepsi ayni dizinde (kok dizininde olmali). Diger bir deyisle davetiye linki ile olusturulan repoda klasor olusturmanizi gerektirecek bir durum yok.
